### ISSUE TITLE
SSTU Tech Balance

### ISSUE THREAD

**Issue by shadowmage45** [shadowmage45/SSTULabs#384]
Except from forums:

----
Started a career. Normal with entry costs set. First flight (R1) was the usual pod+flea+chute+goo+fins. 

Second rocket:

R2 stock: marginal cost 9,602 funds, 30 parts. Mass 15.6t. I think it makes orbit dumping the 1-2 science parts (thermo and barometer) for tanks. Doesn’t orbit as-is (but close). Downside would be lower science return, though. (not that that matters much in stock, you can farm science around KSC, I suppose).

R2SSTU: marginal cost 9,003 funds. 12 parts. Mass just under 18t. Makes orbit with thermo, barometer, and antenna. Could have added goo.

The unlock costs were substantial, however, which made the SSTU one cost 28,603 vs the stock one with entry costs adding up to 14,302.

That seems OK to me, actually, except that future SSTU rockets will then be cheaper, and will have far more capability in terms of payloads, due to all the available part slots.

The first Mun rocket will be more important balance wise I think.

----

 On career balance, the first upgrade is likely the pad, as it is cheap. That makes a huge difference as it increases the mass, and in SSTU you don't need part count. Currently, the tanks in SSTU allow for full length selection from first unlocking them, which is 9*FL-T200. The trick is that I don't like limiting the SSTU lengths, as that defeats the purpose, IMHO. Can the dry mass be too high, then use the "upgrade" functionality to make them sensible?

I think to start, I'd (sadly) move all the SSTU stuff up to at least the 45 level (tier 4) to start. Perhaps the small diameter tank (.625) lower at tier 3, instead of having that diameter be a decrease node bump higher up... The trick is that we need a payload for it. I'd love to see a small probe core---like a nosecone shape---in the 0.625 size, then throw those in early.

Soyuz was really contemporary to Apollo, but unless SSTU adds in a Gemini analog, that will leave a huge gap. I think for gameplay, I'd put the Soyuz spacecraft stuff at the 90 science level. The MFT-D perhaps can stay where it is, as well as the 1.875 diameter change, and I guess the Russian engines as well.

The Soviet station stuff should all be in the 160 tier, I guess. The COS stuff perhaps in the next tier (300 science points, like the Shuttle parts). Inflatables can appear in the 550 tier, and the tori in the 1000 perhaps. I suppose the 2 "A" inflatables could appear with Shuttle in the 300s.

The radial materials science bay should likely move to 90 as it's a miniaturized version of the stock one that appears at 45 science points. Put it in the next node over, "miniaturization."

Solar panels. The first stock panel is at 90, then all the SSTU ones at 160. I'd put all the stuff available on Soviet stations and smaller in the 160 node I guess, and the rest in 300 (High Power Electrics).

When you get Shuttle back, that goes with the rest of the mk3 stuff (the 300 nodes).

I might be inclined to bump the petal adapters to 160 from 90, just to spread things out.

The modular decoupler change to 2.5m needs to drop a node to match the 2.5m tanks, IMO.

So to clarify on tanks, I'd have the SSTU 1.25 start at tier 4 (45 pts), the 1.875 diameter, and soyuz stuff at tier 5 (90 puts), then 2.5m at tier 6, and so on. 

----

 Large super draco, but with half the fuel unloaded so it can actually launch. Goes ~40km. Nose is min tank with nosecone. 8 parts, 4 of which are stock fins.

Solid version. 9 parts, 4 fins, plus I added reaction wheel. Forgot I could gimbal solid, so it should be 8. Made it (just) to space as an IRBM.

----

 So the solid rocket version is pretty effective, and better yet, it's already done.

I'd be inclined to unlock the least capable SRB at 0.625 from the start, along with perhaps probe core at the same diameter, and the MFT-A also at 0.625 (people can configure it as EC for a sounding rocket).

**Comment by taterkerman** [comment]
One slightly unbalancing thing is the ability for all the crew parts to allow crew reports with no crew. Is this a function of a mixture of probe and crew functionality? I generally have assumed that it fills the role of a camera (a science part conspicuously missing from stock KSP), so it's not a concern at that level. Compared to stock, however, it does fulfill various contract requirements, and provides some reasonable, and 100% transmittable science return. As a result it speeds tech tree progression (given how cheap the early nodes are). This mitigates concerns about moving the parts up the tree a little, as the player can use Soyuz parts as unmanned probes as soon as they get them (which is exactly what the Soviet lunar Zond probes were, BTW).

**Comment by taterkerman** [comment]
The Soyuz stuff comes online before solar parts are available at all, which is another reason to move everything up the tech tree slightly. Its a pretty strong incentive to use the Soyuz or SM for everything.

**Comment by shadowmage45** [comment]
Sadly if the SC-A were moved any further down in the tech-tree, it would become useless for its intended purpose (a 2-man capsule to bridge between the stock 1 and 3-man parts).  Moving it further up the tree places it too close to the stock 3-kerbal capsule/sc-b unlock.

I'm about tempted to say screw the stock tech tree altogether.  It makes no sense.  Basic things like solar, fuel cells, ladders, and standard wheels should be available from the beginning.  Size isn't everything (advances should be based on you know... technological advances... not 'bigger is better').

TL:DR -- not my fault stock tech progression doesn't make sense....

**Comment by taterkerman** [comment]
Could not agree more on the tech tree. Been a peeve for a while.

**Comment by taterkerman** [comment]
MST-ISS is too early in the dumb stock tree. It's concurrent with the initial, stock OX-STAT panel (the single fixed panel)

**Comment by shadowmage45** [comment]
Noted; yeah, I need to do a good pass over all the station-core parts; none of them have received any real tech balance yet.

**Comment by taterkerman** [comment]
Yeah, I'm playing a career, and thought I'd post observations as I notice them in terms of how they play out in a  real career.

I'm not sure which one would make sense, but it seems like a Station Core part (or a few) should be in the same node where the Hitchhiker appears. The trouble is that given the solar panels, etc, they are perhaps OP for that node. I suppose that using the upgrade functionality they could appear without the panels as an option if we are talking about DOS parts? Or, (I almost hate to say this) with the stock, OX-STAT as the only available choice? Otherwise, the COS parts, while "realistically" later parts (1980s) are by far the most simple Station Core parts, so perhaps they'd fit in that Hitchhiker node best (maybe the smaller ones).

At some point in the future (Base Core?), I'd think a useful adapter would be 2 new airlock adapters. Instead of having the hatch point along the long axis of the cylindrical part, have them on the side of a simple, cylindrical adapter. One would have the hatch such that if the station part was resting on the "bottom" node, the crew would exit basically on the ground (for use of the COS part as a base part, basically, vertically). The other would have it such that the crew would be upright when the COS part (or whatever) was laying on its side. The later adapter might be asymmetric such that they can step right onto the ground (the hatch could be on the side, or the end, I suppose).

**Comment by shadowmage45** [comment]
You bring up some good points; feel free to post any suggestions you come across while playing through your career game -- more data will be useful when I start on the balancing.

You also just gave me an idea -- I might be able to lock some/all of the StationCore solar panel options behind a PartUpgrade/multiple upgrades.  I -think- that the stock code will allow for patching in of additional nodes as is, or I might be able to come up with a workaround.  Just a thought though -- mostly it would be to fix the tech-tree setup.  The core part can unlock where it logically should (with the hitchhiker, or nearby), and the solar panels could come as upgrades along the path of electric tech nodes.

**Comment by taterkerman** [comment]
Yeah, that could work. The first unlock could be no panels at all, then the player has to stick ugly stock panels on it. Once the good panels unlock, then they get the SSTU stuff. My observation in another issue here about the upgrade functionality is possibly an issue (for people who play with fund costs to unlock). Can upgrades be "free" from a funds standpoint? That might make your life easier in terms of "Why can't I make my tanks X size in career?" standpoint.
